### ISSUE TITLE
Can't modify a feature in annotation tool without create a new point

### ISSUE THREAD

**Issue by FBeneville** [geosolutions-it/MapStore2#8608]
## Description
After creating some line or polygon, when you select a feature back for some modification. You can't do anything, but just add a new point. Only then you can move a point or create a new one starting between two existing point as it is able when you create your feature

## How to reproduce

-  Create a line or polygon: you can move existing point and create new point between two existing point.
-  Create a new feature
- Come back to the first line or polygon. You can't do anything, but just click and create a new point.
- You can now move point or create a new one between two existing point

*Expected Result*

We expect that a selected feature allow the same things that a new one without the need to create a new point

*Current Result*

- [x] Not browser related

FireFox using geosolution platform: https://georchestra.geo-solutions.it/mapstore/#/

## Other useful information
<img width="678" alt="Capture" src="https://user-images.githubusercontent.com/55783032/191936881-71bf60bf-42e4-4e14-8a19-bae0cd93e21b.PNG">

**Comment by tdipisa** [comment]
@ElenaGallo is this still valid?

**Comment by ElenaGallo** [comment]
@tdipisa No, returning to the first line or polygon, I can move the points or create a new one between two existing points. Both in Chrome and Firefox
